src/components/SearchResults/SearchResultItem.js

```diff
@@ -26,12 +25,51 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-function SearchResultItem({ country, title, link, summary, resultType }) {
+function SearchResultItem({ title, resultType, slug, id, countrySlug }) {
   const classes = useStyles();
+  const [profileSlug, setProfileSlug] = useState('');
+
+  const type = ['topic_page', 'profile_page', 'profile'].includes(resultType)
+    ? 'Analysis'
+    : 'Data';
+
+  useEffect(() => {
+    if (resultType === 'topic_page') {
+      fetch(`${config.WP_BACKEND_URL}/wp-json/acf/v3/${resultType}/${id}`).then(
+        response => {
+          if (response.status === 200) {
+            response
+              .json()
+              .then(({ acf: { section_topics: profileSection } }) => {
+                if (profileSection && profileSection.length > 0) {
+                  setProfileSlug(profileSection[0].post_name);
+                }
+              });
+          }
+        }
+      );
+    }
+  }, [resultType, id]);
+
+  const countryFound = config.countries.find(c => c.slug === countrySlug);
+  let link = '';
+  let country = '';
+  if (countryFound) {
+    country = countryFound.name;
+  }
+
+  if (resultType === 'topic_page') {
+    link = `/profiles/${countrySlug}/${profileSlug}#${slug}`;
+  } else if (resultType === 'profile_page') {
+    link = `/profiles/${countrySlug}/${slug}`;
+  } else if (resultType === 'profile') {
+    link = `/profiles/${countrySlug}`;
+  }
```

Comment:
Where do we point `link` for `Data` type?

Reply:
@kilemensi  Data type was onhold as I was waiting for the merge of hurumap /flourish post structure. I will create a story for this and the updates needed on data by topic pages

Reply:
https://www.pivotaltracker.com/story/show/170546625